natrix/main.py

```diff
@@ -0,0 +1,44 @@
+import os
+import glob
+import click
+
+from natrix.utils import parse_file_to_ast
+from rules import rules
+
+def lint_file(file_path):
+    ast = parse_file_to_ast(file_path)
+
+    for rule in rules:
+        rule.rule_fn(ast)
+
+
+def find_vy_files(directory):
+    # Recursively find all .vy files in the given directory
+    return glob.glob(os.path.join(directory, '**', '*.vy'), recursive=True)
+
+@click.command()
+@click.argument('path', required=False, default=None)
```

Comment:
why don't we use the [path type](https://click.palletsprojects.com/en/8.1.x/api/#click.Path)?